test/units/plugins/filter/test_uuid.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible.plugins.filter.core import to_uuid
+
+
+UUID_DEFAULT_NAMESPACE_TEST_CASES = (
+    ('example.com', 'ae780c3a-a3ab-53c2-bfb4-098da300b3fe'),
+    ('test.example', '8e437a35-c7c5-50ea-867c-5c254848dbc2'),
+),
```

Comment:
Unit tests are failing because this is being interpreted by `parametrize` as the first item in the list of test cases, making `value=('example.com', 'ae780c3a-a3ab-53c2-bfb4-098da300b3fe')` and `expected=('test.example', '8e437a35-c7c5-50ea-867c-5c254848dbc2')`
```suggestion
)
```